### DISCUSSION TITLE
Custom signal range for plotBrowserTrack()

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#903] [Questions / Documentation] [answered]
Hi @rcorces and @jgranja24 

Is there a way to plot a custom signal range using the ReadsinTSS Norm method?

I would like to lower the default signal range so that I can more clearly see lower signal peaks. 

Thanks!

**Comment by rcorces** [comment, marked as answer]
The `ylim` argument?

**Reply by RegnerM2015** [reply]
Sorry for missing that one! I will be sure to read more carefully in the future.

**Reply by RegnerM2015** [reply]
@rcorces 

I have tried manipulating the ylim argument, but the browser track becomes blank. Does it matter that I am using ArchR 0.9.5?

```
plot <- plotBrowserTrack(atac,geneSymbol = "SOX9", groupBy = "predictedGroup_ArchR",
                         ylim=.5)

```
![image](https://user-images.githubusercontent.com/54336249/126001982-9d919ea0-a96b-4266-b436-55190542d8af.png)

**Reply by rcorces** [reply]
I think there was a bug in `.bulkTracks()` which I think I fixed via 
https://github.com/GreenleafLab/ArchR/commit/a861399f489fa2a870e42cdf759166d44543caaf
and
https://github.com/GreenleafLab/ArchR/commit/8caa716086832c2560b867b6e4a0d1bab48716e1

You can update this in `0.9.5` on your end. That being said, dont use `ylim=0.5`, that is way too low for a quantile. start at `0.9`.

If you could confirm that this fix works for you, I'll incorporate it into `1.0.2`

**Reply by RegnerM2015** [reply]
a861399 works on my end!

Thanks so much for your help!

**Reply by rcorces** [reply]
For posterity, my changes mentioned above were not correct and actually broke the ArchRBrowser. The problem was actually with the `plotBrowserTrack()` function parameter documentation. I've corrected this in `release_1.0.2` via 63e7883586a013b9a9c264ddb19dea18d9287087
Long story short, the syntax is `ylim = c(0,0.9)` rather than using a single value for `ylim`

This was in response to https://github.com/GreenleafLab/ArchR/issues/1206

**Reply by RegnerM2015** [reply]
Thanks! I have since updated my workflows to the latest version of ArchR so I don't anticipate having these issues.
